Sources/FoundationInternationalization/Formatting/Number/NumberFormatStyleConfiguration.swift

```diff
@@ -332,6 +332,7 @@ public enum CurrencyFormatStyleConfiguration {
         var roundingIncrement: RoundingIncrement?
         var presentation: Presentation
         var notation: Notation?
+        var currencyCode: String
```

Comment:
This would break the backwards compatibility of the `Decodable` conformance, i.e. decoding a currency format style that was encoded by a prior version of foundation would fail. We should probably make this optional and fall back to the locale's currency code if the value is not present here.

Reply:
You're right, thanks. I'll fix that up and mark this ready for review once that's done.

Slightly related, we started requiring explicit `Codable` implementation a while ago, but these types predated that practice. I'm really tempted to do that together here although not sure if that's a good idea 😞